packages/core/test/server/index.test.ts

```diff
@@ -112,20 +113,36 @@ describe('FastStore GraphQL Layer', () => {
   })
 
   describe('VTEX API Extension', () => {
-    it('should return a valid GraphQL schema', async () => {
-      const schema: GraphQLSchema = await getVtexExtensionsSchema()
+    it('should return a valid GraphQL schema', () => {
+      const schema: GraphQLSchema = getVtexExtensionsSchema()
 
       // `assertValidSchema()` will throw an error if the schema is invalid, and
       // return nothing if it is valid. That's why we're checking for `undefined`.
       expect(assertValidSchema(schema)).toBeUndefined()
     })
 
-    it('getTypeDefsFromFolder function should return an Array', async () => {
+    it('getTypeDefsFromFolder function should return an Array', () => {
       const typeDefs = getTypeDefsFromFolder('vtex')
       expect(typeDefs).toBeInstanceOf(Array)
     })
   })
 
+  describe('Third Party API Extension', () => {
+    it('should return a valid GraphQL schema', () => {
+      const schema = getThirdPartyExtensionsSchema()
+
+      // `assertSchema()` will throw an error if the parameter is not a schema
+      // we are using thi9s assertion because initially the Third Party schema
+      // does not have a root Query type so is not a "valid" schema
+      expect(assertSchema(schema)).toBeTruthy()
```

Comment:
hmm, interesting. Do you think it's better we change the other examples to use the `assertSchema` instead of `assertValidSchema`? we could change it in another PR just to keep this PR cleaner.

Reply:
IDK, because the vtex API is already a validSchema, because it has a root Query, and it always should have